meta-intel-edison-distro/recipes-support/oobe/oobe_git.bb

```diff
@@ -14,6 +14,7 @@ RDEPENDS_${PN} = "python3-core python3-bottle"
 
 do_install() {
    install -d ${D}${libdir}/edison_config_tools
+   install -d ${D}/var/lib/edison_config_tools
```

Comment:
Even though this seems a bit strange, duplicated paths, hard coded.

Reply:
I have just restored the old behaviour from Intel. `configure_edison` requires this path to be able to create the mentioned status files.